mappings/markdown/Zeek/v1.3.0/ssh_log/samples/ssh_log.ocsf

```diff
@@ -0,0 +1,126 @@
+{
+  "metadata": {
+    "version": "1.3.0",
+    "logged_time": 1729064044.950981,
+    "loggers": [
+      {
+        "name": "sensor"
+      }
+    ],
+    "log_name": "ssh",
+    "uid": "CwjjT11TKp4IDDkiol",
+    "product": {
+      "name": "Zeek",
+      "vendor_name": "Zeek"
+    }
+  },
+  "category_uid": "4",
+  "class_uid": "4007",
```

Comment:
This is the last remaining validation issue, the value should be changed to an integer.